saleor/static/js/dashboard.js

```diff
@@ -25,8 +25,63 @@ function openModal() {
 }
 
 $(document).ready(function() {
+  var $mainNav = $('#main-nav')
+  var $windowWidth = $(window).width()
+  var $menuToggle = $('.menu-toggle')
+  var $closeMenu = $('#close-menu')
+  var $openMenu = $('#open-menu')
+  function openMenu(a) {
+    $mainNav.animate({
+      'left': '250px'
+    }, a)
+    if ($windowWidth < 1650 && $windowWidth > 991) {
+      $('main .container, .subheader .nav-wrapper').animate({
+        'marginLeft': '250px'
+      }, a)
+    } 
+    $openMenu.addClass('hide')
+    $closeMenu.removeClass('hide')
+    $menuToggle.addClass('fixed')
+    if ( $windowWidth > 990 ) {
+      $.cookie('menu', 'open', { path: '/' })
+    }  
+  }
+  function closeMenu() {
+    $mainNav.animate({
+      'left': '0'
+    })
+    $closeMenu.addClass('hide')
+    $openMenu.removeClass('hide')
+    $menuToggle.removeClass('fixed')
+    if ($windowWidth > 990) {
+      $.removeCookie('menu', { path: '/' })
+    }
+    if ($windowWidth < 1650) {
+      $('main .container, .subheader .nav-wrapper').css({
+        'margin-left': 'auto'
+      })
+    }
+  }
+  $openMenu.click(function() {
+    openMenu(400)
+  })
+  $closeMenu.click(function() {
+    closeMenu()
+  })
+  if ($windowWidth < 991) {
```

Comment:
I'm not a fan of hardcoding the pixel values all around the place. Could we define some constants for break points and use those later in code?

Also how do we handle resizing the window?